rust/lance-index/src/vector/hnsw/builder.rs

```diff
@@ -130,22 +161,21 @@ impl HNSWBuilder {
         for cur_level in self.levels.iter().rev().take(levels_to_search) {
             let candidates = beam_search(cur_level, &ep, vector, self.ef_construction)?;
```

Comment:
this doesn't follow the comments above which passing `ef=1`, any improvement for the modification?